pkg/stub/handler.go

```diff
@@ -588,6 +588,10 @@ func (h *Handler) Handle(event util.Event) error {
 		// Every time we see a change to the Config object, update the ClusterOperator status
 		// based on the current conditions of the Config.
 		cfg = h.refetchCfgMinimizeConflicts(cfg)
+		//TODO remove this setting of version once we start getting samples for z or ppc
+		if len(cfg.Spec.Architectures) > 0 && cfg.Spec.Architectures[0] != v1.AMDArchitecture && cfg.Spec.Architectures[0] != v1.X86Architecture {
```

Comment:
i'm a little confused as to why this is specifically checking array index 0?

1) do we support multiple architectures being specified?
2) if we do, what happens if x86 is listed first and Z is listed second?

Reply:
> i'm a little confused as to why this is specifically checking array index 0?
> 
>     1. do we support multiple architectures being specified?

at the moment, now ... we just have a place holder given this is an array to support it in the future

> 
>     2. if we do, what happens if x86 is listed first and Z is listed second?

we hard code the entry at start up, and the first time through should set all the clusteroperator related bits

Reply:
adding validation type logic that handle a cluster admin replacing our boostrapping with incorrect choices is doable, but more complicated 

where we compare what they set in the array with what the golang arch env vars report, etc.